CakePHP/Sniffs/Formatting/BlankLineBeforeReturnSniff.php

```diff
@@ -63,10 +63,15 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         ) {
             return;
         } elseif (count($prevLineTokens) > 0) {
-            $phpcsFile->addError(
+            $fix = $phpcsFile->addFixableError(
                 'Missing blank line before return statement',
                 $stackPtr
             );
+            if ($fix === true) {
```

Comment:
As per #102 this is not the proper way to check if fixer is supposed to run. It states `$phpcsFile->fixer->enabled === true` be used instead.